subsys/net/ip/dhcpv4.c

```diff
@@ -764,17 +764,8 @@ static bool dhcpv4_parse_options(struct net_pkt *pkt,
 			}
```

Comment:
typo in commit `net: dhcpv4: use dns_resolve_configure() API` subject, s/configure/reconfigure/
same comment to all the other commits in this PR that have similar text

Reply:
Fixed, thanks!